### PR TITLE
RELATED: BB-2069 MVF disable enter when apply is disabled

### PR DESCRIPTION
**Pull request by yanlipnican**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

File: src/components/filters/MeasureValueFilter/DropdownBody.tsx [hunk 1]
```diff
@@ -209,19 +208,25 @@ class DropdownBodyWrapped extends React.PureComponent<IDropdownBodyProps, IDropd
...
             ? { value: value.value * 100 }
             : { from: value.from * 100, to: value.to * 100 };
-        return rawValue;
     };
 
     private onApply = () => {
+        if (this.isApplyButtonDisabled()) {
+            return;
+        }
+
```
Comment (yanlipnican) [inline src/components/filters/MeasureValueFilter/DropdownBody.tsx]: This is the fix, everything else apart from tests is boy scouting

**Comment by qduval** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2977/ : UNSTABLE in 14m 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5798/ : SUCCESS in 8m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1394/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1931/ : SUCCESS in 23m 49s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2978/ : SUCCESS in 15m 39s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5799/ : SUCCESS in 7m 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1395/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1932/ : SUCCESS in 18m 42s

**Pull request merged**